src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/edit.html.twig

```diff
@@ -66,11 +66,9 @@
 
       {% block product_tab_combinations %}
         <div role="tabpanel" class="form-contenttab tab-pane" id="combinations-tab">
-          {{ include('@PrestaShop/Admin/Sell/Catalog/Product/Tabs/combinations.html.twig', {
-            'combinationsForm': combinationsForm,
-            'combinationsLimit': combinationsLimit,
-            'combinationsLimitChoices': combinationLimitChoices,
-          }) }}
+          {{ render(controller('PrestaShopBundle:Admin\\Sell\\Catalog\\Product\\Combination:listForm', {
```

Comment:
This render shoud be moved into `src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Tabs/combinations.html.twig`

It's the template that will contain the whole tab:
- combination list
- combination generator
- attributes list

So the idea of the templates in `Tabs` folder is to have a single file to include in the `edit.html.twig` template to keep it small and easy to read

Then templates in `Tabs` may be composed of different blocks from the `Blocks` folder

Or at least that's the idea I had to keep things organized, but that's probably not the only way and it's challengeable

Reply:
This still needs to be moved in the tab template